pkgs/by-name/pr/presenterm/package.nix

```diff
@@ -2,46 +2,42 @@
 , fetchFromGitHub
 , rustPlatform
 , libsixel
+, nix-update-script
 , testers
 , presenterm
-, stdenv
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "presenterm";
-  version = "0.7.0";
+  version = "0.8.0";
 
   src = fetchFromGitHub {
     owner = "mfontanini";
     repo = "presenterm";
     rev = "refs/tags/v${version}";
-    hash = "sha256-I5L+Wygj9ApQu/5fm55okwNbyxOiF++7BDl765MLnjY=";
+    hash = "sha256-sMhowTXPzZcIOV4Ny9NzvgXGsZSPBJGDg9JvuoZoSUc=";
   };
 
   buildInputs = [
     libsixel
   ];
 
-  cargoHash = "sha256-w1uXCH8Ybf78EPTIKrhPlPHAnNBp1iiBpFJHY98IPWY=";
+  cargoHash = "sha256-2aHJnGSuP0TEBMxF1zljbEyk1g6ECTpnByyH8jaj78s=";
 
-  # Crashes at runtime on darwin with:
-  # Library not loaded: .../out/lib/libsixel.1.dylib
-  buildFeatures = lib.optionals (!stdenv.isDarwin) [ "sixel" ];
```

Comment:
Why not keep that?

Reply:
I was not able to reproduce the bug. Do you happen to own a mac ?

Reply:
I was able to present a slideshow with images on the darwin builder.
Images appear very blurred but I have not experimented any crash.

Reply:
@GaetanLepage works on `aarch64-darwin` in WeTerm, no crashes 
![image](https://github.com/user-attachments/assets/587ae150-61e2-40f3-8b87-e70ada2bd2d1)

Reply:
The removed code added sixel if not on a Mac -- so the removal of that code means non-darwin machines won't have sixel image support, not that Darwin will crash.

Reply:
> I was able to present a slideshow with images on the darwin builder.
> Images appear very blurred but I have not experimented any crash.

This attempt was using `foot`.

Reply:
I guess my statement still stands, I just don’t know of a terminal that only supports sixel off hand.

Reply:
All I'm trying to say but might not be saying it clearly is that originally `sixel` was turned on for all builds; I turned it off conditionally for Darwin (https://github.com/NixOS/nixpkgs/commit/e89f61f917a046375990f56a5f6a713314dd8098) to fix a crash. So undoing my change would mean unconditionally building with sixel support, not deleting the code entirely.

If everyone agrees we should never build with sixel support, that's one thing, but the bug I fixed was related to always building with sixel, not never building with it.

Reply:
Oh right, I completely misread the code, my attempt was to try to re-enable sixel for all platforms (which is not what I have done).

Reply:
Nevermind, it still crashes. I have reverted the patch.